misc/mutual-tls-helper-stuff/README.asciidoc

```diff
@@ -0,0 +1,49 @@
+== Mutual TLS helper utilities
```

Comment:
This whole directory (`mist/mutual-tls-helper-stuff)` should be removed. I put it here just for testing. The final PR doesn't require it (since we have tests).